R/SEIRDAge.R

```diff
@@ -174,7 +174,7 @@ setMethod(
     names(trans_params) = object@transmission_parameter_names
 
     # check format of parameters b, k and g
-    if(length(b) != 1 | length(k) != 1 | length(g) != 1){
+    if(length(b) != 1 | length(k) != 1 | length(g) != 1| length(mu) != 1){
```

Comment:
Can't mu have either 1 element, or alternatively 1 for each age group.
In general, is there a reason not to allow the other parameters to be age structured as well?

Reply:
I think you might have an old version of the branch, because I have that condition added at some point. Again, you are right about mu being dimension 1 here.
I decided to keep this class simple for the time being with no age structure, so it is easier to follow for somebody not knowledgable enough.
I will create a separate SEIRDVAge for the age-structured case.